client/NEWS.txt

```diff
@@ -1,5 +1,8 @@
 NEWS
 
+2.3.1
+- Added support for key overrides in localhost mode
```

Comment:
so here in NEWS the idea is not to repeat what it's in CHANGES but so make it more of a sentences:

`- Adds the ability to include keys to the locamode file to use for matching responses to getTreatment`